doc/whats_new.rst

```diff
@@ -18,6 +18,7 @@ Changelog
 Bug
 ~~~
 
+- Encoding of BrainVision .vhdr/.vmrk files is checked to prevent encoding/decoding errors when modifying, by Dominik Welke (`#155 <https://github.com/mne-tools/mne-bids/pull/155>`_)
```

Comment:
It'd be good if you make your name a link to your github user page / or your personal webpage - see the last part of `whats_new.rst`